### ISSUE TITLE
Use the Notifications API

### ISSUE THREAD

**Issue by netizen-ais** [Natizyskunk/vscode-sftp#171]
**Is this a similar or duplicate feature request?**
- [ ] I don't know. I will go check it.
- [X] No.

**Is your feature request related to a problem? Please describe.**
The status bar can be cluttered, while UX notifications are common and perfect for this task

**Describe the solution you'd like**
Show a [toast notification](https://code.visualstudio.com/api/ux-guidelines/notifications) in addition to (or selectable in extension settings) status bar reports.

**Describe alternatives you've considered**

**Does this project help you?**
- [x] Yes. SFTP IS AWESOME!

**Comment by Natizyskunk** [comment]
@netizen-ais,
Can you be more precise on wich notifications you would like to see in a toast please ?

**Comment by netizen-ais** [comment]
Of course. I forked a similar package for the Atom IDE that was idle for a long time. I added that feature and the notifications I did implement were protocol+file with local/remote paths, and process result.

I didn't take the time to inspect vscode's API, but I'd live a similar (maybe biassed) approach

**Comment by Natizyskunk** [comment]
Would you actually be ok to do add it to the extension and create a pull-request ?

**Comment by yCodeTech** [comment]
@Natizyskunk I've just configured a dev and prod profiles, and I'm using an Elgato Stream Deck with vscode integration. I have set up a button to switch to a specific profile when pressed (using the args). I would love for there to be a toast notification of when a profile changes, letting me know of the new profile. I think this would be the perfect usage of toast notification.

Please can you make this happen, thanks.
